psi4/src/psi4/libmints/irrep.cc

```diff
@@ -128,13 +128,12 @@ void IrreducibleRepresentation::init(int order, int d, const char *lab, const ch
     else
         csymb = nullptr;
 
-    if (rep) {
-        delete[] rep;
-        rep = nullptr;
+    if (rep.size()) {
```

Comment:
Perhaps I missed this, but how can be sure that `rep` exists here?

Reply:
C++ won't allow things that don't exist. `rep` is a length-zero vector by default.